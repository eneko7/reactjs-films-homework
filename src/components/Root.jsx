import { hot } from 'react-hot-loader';
import App from './App/App.jsx';

export default hot(module)(App);