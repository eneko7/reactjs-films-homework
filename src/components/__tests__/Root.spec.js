import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Root from '../Root.jsx';

describe('Root', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Root />);
    it('renders correctly', () => {
        const result = renderer.getRenderOutput();
        expect(result).toMatchSnapshot();
    });
});