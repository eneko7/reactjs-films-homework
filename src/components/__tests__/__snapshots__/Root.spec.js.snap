// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root renders correctly 1`] = `
<div
  className="main_container"
>
  <Header />
  <Footer />
</div>
`;
